package.json

```diff
@@ -39,18 +39,24 @@
   },
   "dependencies": {
     "@blockworks-foundation/mango-client": "3.4.7",
-    "@project-serum/anchor": "0.24.2",
-    "@solana/spl-token": "0.1.8",
-    "@solana/web3.js": "1.42.0",
     "camelcase": "5.3.1"
   },
+  "peerDependencies": {
+    "@project-serum/anchor": "0.24.2",
+    "@solana/spl-token": "0.2.0",
```

Comment:
Not sure we want to use that version, we had kept 0.1.8 on purpose so far

Reply:
Yes downgraded to `0.1.8`! check the comments on `uxd-staking` PR